Diplom 2

Penchtest opened the pull request:
Использованы библиотеки JUnit, RestAssured, Allure, Datafaker. Написаны тесты на создание пользователя, проверку логина, изменение данных пользователя, удаление пользователя; создание заказа, запрос заказов

Penchtest committed:
Diplom 2

nikolay-977 commented on src/main/java/site/nomoreparties/stellarburgers/generator/IngredientsGenerator.java:
```diff
@@ -0,0 +1,73 @@
+                .then()
+                .extract()
+                .path("data");
+
```
Code Style. Убери лишние пропуски строк. Для зрительного разделения кода достаточно использовать один пропуск строки. Между фигурными скобками пропуски не нужны. Пропуски строк должны быть однообразными.

nikolay-977 commented on src/main/java/site/nomoreparties/stellarburgers/requests/OrderRequest.java:
```diff
@@ -0,0 +1,41 @@
+import static io.restassured.RestAssured.given;
+
+public class OrderRequest extends RestRequestModel {
+    public ValidatableResponse createOrderWithoutAuth(OrderRequestBody orderRequestBody) {
```
При реализации шагов используй аннотацию @Step("Создание заказа"). Без аннотации @Step шаг не отобразится в Allure отчете. В скобках указывай описание шага. Это сделает отчет Allure информативнее, что позволит сократить время локализации дефекта.

Penchtest committed:
Исправлены замечания

Убраны лишние переносы строк, добавлены аннотации @Step, убраны лишние
шаги в тесте на изменение данных без авторизации
